pkg/commands/clone/git.go

```diff
@@ -81,3 +92,51 @@ func rawClone(repo api.Repo, path string) utils.CtxErr {
 
 	return utils.CtxErr{}
 }
+
+// Raw function for cloning the repo
+func sshClone(repo api.Repo, path string) utils.CtxErr {
```

Comment:
Would you mind making this a generic function that just passes in the clone URL instead of essentially duplicating a  bunch of code? All of the functionality is in the `rawClone` function